internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -430,6 +430,28 @@ func resourceKubernetesClusterNodePoolSchema() map[string]*pluginsdk.Schema {
 			Type:     pluginsdk.TypeBool,
 			Optional: true,
 		},
+
+		"security_profile": {
```

Comment:
Please also check if these properties can be set in the kubernetes cluster's default node pool.